apps/researcher/src/messages/en/messages.json

```diff
@@ -158,7 +158,14 @@
     "objectListsSubTitle": "This community has {count, plural, =0 {0 lists} =1 {1 list} other {# lists}}",
     "membersTitle": "Members",
     "error": "Something went wrong while fetching the community. Please try again later.",
-    "addObjectListButton": "Add list"
+    "addObjectListButton": "Add list",
+    "editDescriptionButton": "Edit description",
+    "labelDescription":"Description",
+    "changeDescriptionSaveButton": "Save",
+    "changeDescriptionCancelButton": "Cancel",
+    "descriptionSuccessfullyEdited": "Description has been updated",
```

Comment:
Or: `descriptionSaved` or `descriptionUpdated`? The edit could have been successful, but it's actually about storing the description that has been edited?